common/src/tflite.cc

```diff
@@ -248,7 +248,7 @@ void tflite_classify() {
   }
   uint32_t end = perf_get_mcycle();
 #ifndef NPROFILE
-  profiler->Log();
+  profiler->LogCsv();
 #endif
   perf_print_all_counters();
```

Comment:
Please also move perf_print_all_counters() inside the #if so performance counters are not printed if profiling has been turned offf.

Reply:
Ahh I see. They are always zero if NPROFILE is set because all the code which touches perf counters is compiled out.